### PR TITLE
Hotfix/suit 9844 Update error info logs

### PR DESCRIPTION
**Pull request by givehug**
Improved real time error logs.

### CODE REVIEW

File: lib/utils/socketErrorMessages.js [hunk 1]
```diff
@@ -530,6 +553,14 @@ module.exports = {
 		errorMap,
 		...payload,
 	}),
+	getInfoErrorMessage: (message, prefix, res, stack) => {
```
Comment (JeKachan) [inline lib/utils/socketErrorMessages.js]: Please add jsDoc here.

File: lib/utils/unusedExpressionWatchers.js [hunk 2]
```diff
@@ -10,20 +9,15 @@ const texts = require('./../texts');
 const unusedLeaves = new Set();
 /**
  * @description data about leaves code location
- * @type {Map<Object, {line: string, file: string}>}
+ * @type {Map<string>}
```
Comment (JeKachan) [inline lib/utils/unusedExpressionWatchers.js]: This is invalid type for Map object, you should use Map<TKey, TValue> or Map.<TKEy, TValue>

File: lib/utils/unusedExpressionWatchers.js [hunk 3]
```diff
@@ -10,20 +9,15 @@ const texts = require('./../texts');
...
  * @param chain
  */
-const registerLeaf = chain => {
-	const parsedStack = stackTraceParser(new SuitestError());
-
-	unusedLeavesData.set(chain, {
-		line: parsedStack[0].line,
-		file: parsedStack[0].file,
-	});
+const registerLeaf = (chain, stack = '') => {
```
Comment (JeKachan) [inline lib/utils/unusedExpressionWatchers.js]: stack is missed in JsDoc

File: lib/utils/socketErrorMessages.js [hunk 4]
```diff
@@ -43,7 +45,7 @@ function defaultMessage({chainData, toString, response}) {
 		: '';
 
 	return response.errorType ?
-		`${response.errorType}: "${toString(chainData)}"${errors}` :
+		`${response.errorType}: "${appendDot(toString(chainData))}"${errors}` :
```
[the remark this answers is not in the document]
Comment (givehug) [reply lib/utils/socketErrorMessages.js]: I would rather append dot to chains descriptions, if its fine with everyone else.

[the remark this answers is not in the document]
Comment (JeKachan) [reply lib/utils/socketErrorMessages.js]: or you can just wrap getErrorMessage in exports section

**Comment by givehug** [comment]
There is one code climate issue left which can't be resolved:
`module.exports = {...` - Similar blocks of code found in 3 locations.
Could be cached.

File: lib/utils/unusedExpressionWatchers.js (hunk 3 above)
Comment (givehug) [reply lib/utils/unusedExpressionWatchers.js]: added

**Pull request merged**